### PR TITLE
Break even per project

### LINKED ISSUE #3667: Break Even per projects
This issue proposes to add the focus of the main expense centers to the projects, with a view to generating a break-even ratio per project.

### PR DESCRIPTION
**Pull request by lomamech**
- Add optionnal property project_id in Fee Center table
- Update Integration and E2E test for Fee Center
- Update User Interface for Break Even Per Project
- Update User Interface for Break Even Per Project
- refactor report breakeven for get the result by project

closes #3667

### CODE REVIEW

**Comment by kwilu** [comment]
Hello @lomamech, it seems like you have referenced #3667 in your pull request description, but you have not referenced them in your commit message description(s). When you reference an issue in a commit message, it automatically closes the corresponding issue when the commit is merged.

Please run `git commit --amend` in your command line client to amend your commit message description with `Fixes #3667.`.

An example of a correctly-formatted commit:
```
commit fabd5e450374c8dde65ec35f02140383940fe146
Author: kwilu
Date:   Sat Mar 18 13:42:40 2017 -0700

    pull requests: Check PR commits reference when issue is referenced.

    Fixes #51.
```

Thank you for your contributions to BHIMA!

**Comment by lomamech** [comment]
@mbayopanda  can you review this PR

File: server/controllers/finance/reports/break_even/break_even_project.js [hunk 1]
```diff
@@ -0,0 +1,282 @@
+function configuration(data) {
```
Comment (mbayopanda) [inline server/controllers/finance/reports/break_even/break_even_project.js:31]: Is it possible to add a description of this function, to help developers to know the importance and use of this function.

File: server/controllers/finance/reports/break_even/report_project.handlebars [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+              </tr>
+              {{/if}}
+            </tbody>
+            <tfoot>
+              {{#if data.cantCalculateBreakEven}}
+              <tr style="width: 40%; font-size: 15px; text-align: center;" colspan="3" scope="row">
+                <td>
+                  <strong>
+                    {{translate "TABLE.COLUMNS.BREAK_EVEN.DIFFERENTIAL_INCOME_STATEMENT" }} <SUB> (1) </SUB>
+                  </strong>
```
Comment (mbayopanda) [inline server/controllers/finance/reports/break_even/report_project.handlebars:279]: Could you add in the bottom of the document the meaning or definition of `(1)` and `(2)`

**Comment by lomamech** [comment]
@mbayopanda  all test are passed

**Pull request merged**